packages/labs/src/components/popover/popover2.tsx

```diff
@@ -383,7 +383,7 @@ export class Popover2 extends AbstractComponent<IPopover2Props, IPopover2State>
 
     private updateDarkParent() {
         if (!this.props.inline && this.state.isOpen) {
-            const hasDarkParent = this.targetElement && this.targetElement.closest(`.${Classes.DARK}`) != null;
+            const hasDarkParent = this.targetElement !== null && this.targetElement.closest(`.${Classes.DARK}`) != null;
```

Comment:
Oh, you'll want to do double-equals (`!=`) so that both `undefined` and `null` satisfy the condition.